QUARKUS-343.md

```diff
@@ -0,0 +1,48 @@
+# QUARKUS-343 - Quarkus and non JSON payload in REST endpoints
+
+## Scope of the testing
+The test scope includes validating Quarkus's capability to handle non-JSON payloads (XML, YAML, and images) efficiently in REST endpoints.
+Tests will cover both JVM and Native mode executions.
+#### Test scenarios
+
+        1. XML Payload Scenario:
+            * Verify the endpoint's ability to consume and produce XML payloads.
+            * Send a request with an XML payload to the endpoint.
+            * Validate the response, ensuring it matches the expected XML structure.
+    
+        2. YAML Payload Scenario:
+            * Check the endpoint's ability to return YAML from a YAML file.
```

Comment:
Endpoint never _returns YAML from a YAML file_. It returns payload you return from method.

If you want to say you load YAML from file and pass it as string / DTO / byte arr whatever, please just it clearly.